themes/pom.xml

```diff
@@ -131,6 +86,72 @@
     </build>
 
     <profiles>
+        <profile>
+            <id>account2</id>
+            <activation>
+                <!-- 'activeByDefault' will not work here as other profiles in this POM are always active, so we need to have a workaround to activate this profile. -->
+                <file>
+                    <exists>.</exists>
+                </file>
+           </activation>
```

Comment:
```suggestion
            <activation>
                <property>
                    <name>!skipAccount2</name>
                </property>
           </activation>
```

Reply:
This allows de-activating account2 generation with:

    mvn install -DskipAccount2

Reply:
Awesome, I really like this approach. Added your suggestion :+1: